src/platform/forms/tests/save-in-progress/01-sip-load-fail.cypress.spec.js

```diff
@@ -53,7 +53,14 @@ describe('SIP Load Fail Test', () => {
     cy.get('body');
 
     cy.intercept('GET', '/v0/in_progress_forms/XX-123', {
-      body: {},
+      body: {
+        errors: [
+          {
+            code: 404,
+            status: 404,
```

Comment:
This is what I'm referring to - I don't see why we need to duplicate the actual response status code inside of the response's body. This seems like a recipe to get a response with a status of 200 that might be considered an "error" if its body indicates so.

Reply:
@bkjohnson first I wanted to say thank you for taking the time to review my code.  I think because I was mocking `fetch` I wrote it this way initially so that I could get the intended status codes.  I decided to replace the custom `fetch` with `msw` which allowed me to actually see the results of a real response (way more helpful 😅 ).  I have updated the code to look at the `status` codes of the response instead.  Thank you again!